src/views/bridge-confirmation/components/bridge-button/bridge-button.view.tsx

```diff
@@ -31,7 +31,7 @@ export const BridgeButton: FC<BridgeButtonProps> = ({
       case "pending": {
         return (
           <Button onClick={onApprove}>
-            {`Allow Polygon zkEVM Bridge to spend my ${token.symbol}`}
+            {`Allow Bridge to spend my ${token.name}`}
```

Comment:
I think we should use token.symbol, because name not so popular
eg USDT name - Tether Token